org.eclipse.jdt.ui.tests.refactoring/.project

```diff
@@ -19,8 +20,14 @@
 			<arguments>
 			</arguments>
 		</buildCommand>
+		<buildCommand>
```

Comment:
Also explain this change or remove it if has nothing to do with your enabling jdt.ls.

Reply:
Removed